config/settings.yml

```diff
@@ -1,13 +1,17 @@
 stacks:
   storage_root: /stacks
-  driver: djatoka
+  djatoka_url: 'http://localhost:8080/resolver'
+  driver: remote_iiif
   djatoka:
+    image: DjatokaImage
```

Comment:
shouldn't these be in `djatoka.image.implementation` and ``djatoka.info.implementation`?